Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -10,13 +10,14 @@ final class ChangePasswordViewController: UIViewController {
   @IBOutlet fileprivate weak var currentPasswordLabel: UILabel!
   @IBOutlet fileprivate weak var currentPasswordTextField: UITextField!
   @IBOutlet fileprivate weak var validationErrorMessageLabel: UILabel!
+  @IBOutlet fileprivate weak var messageBannerContainer: UIView!
```

Comment:
Maybe `messageBannerViewControllerContainer`?

Reply:
I've actually decided to use protocol extension to make the code more DRY and with that I think it's more appropriate for the protocol to have its variable be called `messageBannerViewControllerContainer`. In order to prevent from naming collisions I've decided to keep `messageBannerContainer` the same as it's now (mostly for clarity and simplicity) and will be wrapping it in the protocol like:
```
extension ChangeEmailViewController: MessageBannerViewControllerDelegate {
  var messageBannerViewControllerContainer: UIView {
    return self.messageBannerContainer
  }
}
```

Reply:
But why wouldn't the `ChangeEmailViewController` in this case just call its container `messageBannerViewControllerContainer` and in that way it's already conforming to the protocol?

Reply:
I couldn't make the compiler happy...it's probably something to do with the fact that `messageBannerViewControllerContainer` is `IBOutlet weak` 🤷‍♂️

Reply:
The only way I was able to do this was to declare protocol as `@objc`:

```
@objc internal protocol MessageBannerViewControllerDelegate {
  var messageBannerContainer: UIView { get }
}
```
and then declare the `IBOutlet` as:
```
@IBOutlet internal var messageBannerContainer: UIView!
```
or by default
```
@IBOutlet var messageBannerContainer: UIView!
```

which causes SwiftLint to produce `Private Outlets Violation` warning 🤷‍♂️ 

This doesn't seem very safe to me and it seems more like fighting the framework so I'd prefer to keep it this way (for now).

Reply:
Sounds good!